azkaban-common/src/main/java/azkaban/executor/container/watch/AzPodStatusExtractor.java

```diff
@@ -366,10 +366,12 @@ private boolean checkForAzPodAppContainerStarting() {
       logger.debug("ContainerStarting false as container status is null or empty");
       return false;
     }
+    // All containers shouldn't be waiting and container should be ready
```

Comment:
This should probably say:
`All containers should be waiting and ContainersReady condition should not be true`

Otherwise the change to the condition itself seems reasonable. Would be good to know more about the sequence of events which led to a problem.